### PR TITLE
Fix for Apple Silicon

### PR DESCRIPTION
**Pull request by josh-allan**
This is a slight patch to the readme to include instructions to provide a workaround for installing `xmlsec` on Apple silicon as this presently fails ungracefully.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +96,21 @@ To pull down the latest version:
 
 ---
 
+### Workaround for Apple M Chips
+
+There is presently an issue with `libxmlsec1` versions => 1.3.0 and Apple Silicon which results in a required dependency `lxml` when `cpython` attempts to compile it, resulting in an ungraceful failure. The following workaround has been tested successfully with a Macbook Pro M1 Pro and Python 3.10+. This involves downgrading `xmlsec` to a known working version (`1.2.37`).
```
Comment (clegaspi) [inline README.md:101]: The first sentence is worded a little awkwardly. Maybe:

There is presently an issue with `libxmlsec1` versions => 1.3.0 and Apple Silicon which results in an ungraceful failure in a required dependency `lxml` when `cpython` attempts to compile it.

**Review by clegaspi — CHANGES_REQUESTED** [review]
Grammar update

File: README.md (hunk 1 above)
Comment (josh-allan) [reply README.md]: Updated in https://github.com/clegaspi/saml_reader/pull/83/commits/48963a6769af310021def18502570edba8efb7d0

**Review by clegaspi — APPROVED** [review]
LGTM

**Pull request merged**